configs/service.yaml

```diff
@@ -23,5 +23,7 @@ grpc_keepalive_timeout: "20s"
 # The following enables OpenCensus Tracing via Cloud Trace
 # for the Datastore client library.
 # It is EXPERIMENTAL and subject to change or removal without notice.
```

Comment:
Can you please remove this comment? This was added for Cloud Trace before we replaced it with Open Telemetry.